robot_player/dxl/dxl_control_table.py

```diff
@@ -602,13 +602,27 @@ class DXLPRO:
                        H42_20_S300_R: 3.4*8250.0/2048.0000,
                       }
 
+    #torque_equation_dict format:
+    #Model number of motor: [slope of method 1,y-intercept of method 1, slope of method 2, y-intercept of method 2
+    #dxl counts to Amps, Amps to dxl counts].Where Method 1 - linear relationship between torque and
+    #current when the motor is stalled at increasing weight; Method 2 - linear relationship between torque and
+    #current when the motor is moving dynamically at increasing weight
+    torque_equation_dict = {
```

Comment:
```suggestion
    TORQUE_EQUATION_DICT = { 
```

Probably use all CAPS for torque_equation_dict so that it matches the style in the rest of the class of using all caps for the class attributes since each DXL class is basically just a giant set of constants